management/branding_test.go

```diff
@@ -3,120 +3,70 @@ package management
+	assert.NoError(t, err)
+	assert.IsType(t, &Branding{}, branding)
+}
 
-	t.Run("Update", func(t *testing.T) {
-		err = m.Branding.Update(&Branding{
-			Colors: &BrandingColors{
-				Primary:        auth0.String("#ea5323"),
-				PageBackground: auth0.String("#000000"),
+func TestBrandingManager_Update(t *testing.T) {
+	expected := &Branding{
+		Colors: &BrandingColors{
+			Primary: auth0.String("#ea5323"),
+			PageBackgroundGradient: &BrandingPageBackgroundGradient{
+				Type:        auth0.String("linear-gradient"),
+				Start:       auth0.String("#000000"),
+				End:         auth0.String("#ffffff"),
+				AngleDegree: auth0.Int(35),
 			},
-			FaviconURL: auth0.String("https://mycompany.org/favicon.ico"),
-			LogoURL:    auth0.String("https://mycompany.org/logo.png"),
-			Font: &BrandingFont{
-				URL: auth0.String("https://mycompany.org/font.otf"),
-			},
-		})
-		if err != nil {
-			t.Error(err)
-		}
-
-		branding, _ = m.Branding.Read()
-		t.Logf("%v\n", branding)
-
-		t.Run("BrandingColors", func(t *testing.T) {
-			err = m.Branding.Update(&Branding{
-				Colors: &BrandingColors{
-					Primary: auth0.String("#ea5323"),
-					PageBackgroundGradient: &BrandingPageBackgroundGradient{
-						Type:        auth0.String("linear-gradient"),
-						Start:       auth0.String("#000000"),
-						End:         auth0.String("#ffffff"),
-						AngleDegree: auth0.Int(35),
-					},
-				},
-				FaviconURL: auth0.String("https://mycompany.org/favicon.ico"),
-				LogoURL:    auth0.String("https://mycompany.org/logo.png"),
-				Font: &BrandingFont{
-					URL: auth0.String("https://mycompany.org/font.otf"),
-				},
-			})
-			if err != nil {
-				t.Error(err)
-			}
-
-			branding, _ = m.Branding.Read()
-			t.Logf("%v\n", branding)
-		})
-	})
-
-	c := &CustomDomain{
-		Domain: auth0.Stringf("%d.auth.uat.alexkappa.com", time.Now().UTC().Unix()),
-		Type:   auth0.String("auth0_managed_certs"),
-	}
-
-	err = m.CustomDomain.Create(c)
-	if err != nil {
-		t.Fatal(err)
+		},
+		FaviconURL: auth0.String("https://mycompany.org/favicon.ico"),
+		LogoURL:    auth0.String("https://mycompany.org/logo.png"),
+		Font: &BrandingFont{
+			URL: auth0.String("https://mycompany.org/font.otf"),
+		},
 	}
 
-	t.Cleanup(func() {
-		m.CustomDomain.Delete(c.GetID())
-		m.Branding.DeleteUniversalLogin()
-	})
-
-	t.Run("SetUniversalLogin", func(t *testing.T) {
-		body := `<!DOCTYPE html><html><head>{%- auth0:head -%}</head><body>{%- auth0:widget -%}</body></html>`
-
-		ul := &BrandingUniversalLogin{
-			Body: auth0.String(body),
-		}
+	err := m.Branding.Update(expected)
+	assert.NoError(t, err)
 
-		err = m.Branding.SetUniversalLogin(ul)
-		if err != nil {
-			t.Fatal(err)
-		}
+	actual, err := m.Branding.Read()
+	assert.NoError(t, err)
+	assert.Equal(t, expected.GetColors().GetPrimary(), actual.GetColors().GetPrimary())
+	assert.Equal(t, expected.GetFont().GetURL(), actual.GetFont().GetURL())
+	assert.Equal(t, expected.GetFaviconURL(), actual.GetFaviconURL())
+}
```

Comment:
Any way to unset or cleanup those values?

Reply:
[6fcbd2d](https://github.com/auth0/go-auth0/pull/50/commits/6fcbd2d49254b9c3c0894e89219d9bb14838da38) Good point! Thanks for spotting this!